content/en/watchdog/faulty_deployment_detection.md

```diff
@@ -19,6 +19,8 @@ Whenever a faulty deployment is detected, Watchdog adds this as an event in the
 
 You can also enable the monitor by clicking the **Suggested Monitors** button, and then **Enable**. The Suggested Monitors button is only available if the service does not yet have a monitor configured. If the button is not available, follow the instruction above to create the monitor from the [New Monitors][3] page.
 
+Since each deployment is analyzed repeatedly, in order to prevent re-alerting multiple time on the same faulty deployment, we recommend setting a recovery time of 60 min for the monitor.
```

Comment:
```suggestion
Each deployment is repeatedly analyzed. To prevent re-alerting of the same faulty deployment, Datadog recommends setting a recovery time of 60 min for the monitor.
```